src/content/docs/workflows/about-workflows/index.mdx

```diff
@@ -0,0 +1,44 @@
+---
+page_id: 85c135db-f55a-4edf-8d1b-0af4140c1925
+title: About workflows
+sidebar:
+  order: 1
+relatedArticles:
+  - d672fca7-6c6a-49f8-85c9-6e86ce99f440
+  - 8554d648-fc2f-4ef9-aaa9-4f27b3c05205
+  - d360c480-49a8-47d3-b178-1abd34ae669f
+---
+
+In Kinde, workflows exist to give you (almost) unlimited power over how Kinde works for you. 
+
+Develop code in your own repo, and push it to a Kinde workflow, where we execute it in ways that you define, triggered by Kinde events. 
+
+Currently, workflows are limited to the following triggers:
+
+- `user:post_authentication` fires after the user has completed single factor authentication (e.g. email + password or Google), and before authorization.
+- `m2m:token_generation` - an m2m transaction fires.
+- `user:new_password_provided` - a user creates or resets a password.
+- `user:existing_password_provided` - a user signs in with their password.
+
+Coming soon:
+- `user:pre_mfa` fires after the user has completed single factor authentication (e.g. email + password or Google) and determined which organization (if any) they are trying to access.
+- `user:tokens_generation` - An ID or access token is generated after authentication or refresh token request.
+
+We will add more triggers (and allow you to create your own) as we develop the feature.
+
+## When to use a workflow
+
+- You want Kinde to do something that it doesn’t currently do.
+- You want to customize an access or ID tokens in ways that we don’t support out of the box. For example, you want to fetch data from an external system to include it as claims in your tokens.
+- You want to reduce the size and weight of tokens, remove things that you don’t need.
+- You want to change how info is rendered, for example, deliver payload via string and not array.
```

Comment:
Want to change the data type of a claim. For example changing from an array to a space delimited string